client/clip_client/client.py

```diff
@@ -115,10 +115,18 @@ def encode(self, content, **kwargs):
         )
         results = DocumentArray()
         with self._pbar:
-            self._client.post(
-                **self._get_post_payload(content, kwargs),
-                on_done=partial(self._gather_result, results=results),
-            )
+            try:
```

Comment:
I just want to confirm will we have the same behavior during `aencode()` and `arank()`

Reply:
no worry, working on it